save-backend/src/main/kotlin/org/cqfn/save/backend/configs/WebSecurityConfig.kt

```diff
@@ -34,8 +44,16 @@ class WebSecurityConfig {
             // FixMe: Properly support CSRF protection https://github.com/diktat-static-analysis/save-cloud/issues/34
             csrf().disable()
         }
-        .formLogin()
-        .and().build()
+        .httpBasic()
+        .authenticationManager(authenticationManager)
+        .and().exceptionHandling {
+            it.authenticationEntryPoint(
+                HttpStatusServerEntryPoint(HttpStatus.UNAUTHORIZED)
+            )
+        }
+        .logout().disable()
```

Comment:
why logout disabled?

Reply:
Because in backend spring-security is only used to populate context with users, all actual login/logout logic is only on gateway. So logout option is not needed here